tests/test_main.py

```diff
@@ -104,6 +104,33 @@ def test_main_stdin_no_changes(capsys):
     assert err == ""
 
 
+@pytest.mark.parametrize(
+    "fixer_args,changed",
+    [
+        ([], True),
+        (["--fixer", "other"], False),
+        (["--fixer", "queryset_paginator"], True),
+        (["--fixer", "queryset_paginator", "--fixer", "another"], True),
+    ],
+)
+def test_main_select_fixer(capsys, fixer_args, changed):
+    input_ = "from django.core.paginator import QuerySetPaginator\n"
+    stdin = io.TextIOWrapper(io.BytesIO(input_.encode()), "UTF-8")
+
+    with mock.patch.object(sys, "stdin", stdin):
+        result = main(["-"] + fixer_args)
+
+    out, err = capsys.readouterr()
+    assert err == ""
+
+    if changed:
+        assert result == 1
+        assert out == "from django.core.paginator import Paginator\n"
+    else:
+        assert result == 0
+        assert out == input_
```

Comment:
I don't like conditional tests, it makes the test code too complicated. can we instead have two test functions, one, for the "changes expected case", one for the "no changes expected"?